README.md

```diff
@@ -7,3 +7,5 @@ UI-based tool to allow any user to visualize polygon or point-based POI data on
 3. Install make from here: https://formulae.brew.sh/formula/make
 4. Create a new .env file and add MAPBOX_ACCESS_TOKEN there
 5. `make run`
+6. install node 16.14.2 (includes npm 8.5.0)
+7. create folder app using command 'npx create-react-app app'
```

Comment:
The development section can be split up into two parts, one for tilesets another for react app.

Mention steps required to run the app and not set up. 
For example here Line number 7 is for setting up a new react app, that we have done already, a dev working on this won't need to do the same. Instead, they might have to just install the correct version of node and then do npm install and then npm start maybe.